README.md

```diff
@@ -1,8 +1,7 @@
 UIP Is Pretty
 =============
 
-UIP Scrapes wallpapers from reddit and applies them as wallpapers on a
-gnome Desktop according to a schedule.
+UIP Scrapes wallpapers from reddit and applies wallpaper on your desktop. Works with windows, mac and gtk based desktops on Linux according to a schedule.
```

Comment:
ahh sorry  i guess i wasn't clear in the issue but the `according to a schedule` part needs to be in the previous sentence. like
`UIP Scrapes wallpapers from reddit and applies them on your desktop according to a schedule. Works with windows, mac and gtk based desktops on Linux.`